mnist/README.md

```diff
@@ -216,51 +136,56 @@ argo submit model-train.yaml -n ${NAMESPACE} --serviceaccount tf-user \
 Your training workflow should now be executing.
 
 You can verify and keep track of your workflow using the argo commands:
+
 ```
-$ argo list
+$ argo -n ${NAMESPACE} list
 NAME                STATUS    AGE   DURATION
 tf-workflow-h7hwh   Running   1h    1h
 
-$ argo get tf-workflow-h7hwh
+$ argo -n ${NAMESPACE} get tf-workflow-h7hwh
 ```
 
-## Monitoring
+After the STATUS to `Succeeded`, then you can use it.
 
-There are various ways to monitor workflow/training job. In addition to using `kubectl` to query for the status of `pods`, some basic dashboards are also available.
 
-### Argo UI
+## 9.Submit serving workflow[optional]
 
-The Argo UI is useful for seeing what stage your worfklow is in:
+**NOTE: Please only run this when you disable model serving in step 7.**
 
 ```
-PODNAME=$(kubectl get pod -l app=argo-ui -n${NAMESPACE} -o jsonpath='{.items[0].metadata.name}')
-kubectl port-forward ${PODNAME} 8001:8001
+argo -n ${NAMESPACE} list # get the workflow name
+WORKFLOW=<the workflow name>
+argo submit model-deploy.yaml -n ${NAMESPACE} -p workflow=${WORKFLOW} --serviceaccount=tf-user
 ```
 
-You should now be able to visit [http://127.0.0.1:8001](http://127.0.0.1:8001) to see the status of your workflows.
-
-### Tensorboard
+## 10.Using Tensorflow serving
 
-Tensorboard is deployed just before training starts. To connect:
+### Install client requirements
 
 ```
-PODNAME=$(kubectl get pod -l app=tensorboard-${JOB_NAME} -o jsonpath='{.items[0].metadata.name}')
-kubectl port-forward ${PODNAME} 6006:6006
+apt install python-pip python-setuptools --no-install-recommends
+pip install -r requirements.txt
 ```
 
-Tensorboard can now be accessed at [http://127.0.0.1:6006](http://127.0.0.1:6006).
+### Web client: Mnist Digit Reader
+
+```
+cd mnist-webapp
+SERVICE_IP=$(kubectl -n ${NAMESPACE} get service -l app=mnist-${JOB_NAME} -o jsonpath='{.items[0].spec.clusterIP}')
```

Comment:
Ho are you connecting to the service? Did you mean to get rid of the port-forward command?

Reply:
yes, get rid of the port-forward command, all things are done inside the server.

It is a bit more difficult for beginners to configure and run kubectl on their desktop/laptop.

Reply:
Why do they need to cd to the directory mnist-webapp?
How can you run the port-forward command inside the server?

Reply:
the server is the node that deployed k8s cluster, they need download this example to server. I think all people can know this without tips.

Reply:
I'm a bit lost. Isn't the server deployed inside Kubernetes?

Reply:
I will remove this webapp, and submit new PR later to include this webapp. Reduce the PR size.

Reply:
Ok thanks.